src/components/researcher/dashboard/Map.jsx

```diff
@@ -5,15 +5,21 @@ import {
   MdOutlineNetworkWifi1Bar,
 } from "react-icons/md";
 import Status from "./Status";
+import { useSession } from "next-auth/react";
 
 const Leaflet = dynamic(() => import("../../Map"), { ssr: false });
 
 const Map = () => {
+  const { data: session } = useSession();
+  const userName = session?.user?.name;
```

Comment:
this is under a protected page so we dont have to do the ?, we can directly grab it with session.user.name